content/topics/solo-learn/javascript/intro-to-javascript/1-welcome-to-javascript/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+_db_id:
+content_type: topic
+ready: true
+tags:
+- javascript
+- solo-learn
+title:Solo Learn - Intro to JavaScript - 1 Welcome to JavaScript

```

Comment:
This will cause an error: colon must be space separated. Run  linter to avoid such issues and test the site locally using Hugo https://github.com/Umuzi-org/ACN-syllabus#to-get-yourself-set-up-on-a-debian-based-machine-linux-ubuntumint

Reply:
Ah sorry, missed that 1 space.